Contuity.xcodeproj/xcuserdata/lexispikerman.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please untrack xcuserdata files, they clutter the repo and aren't really necessary (xcshared stuff is fine)